src/app/components/core/dashboard/dashboard.component.ts

```diff
@@ -272,6 +276,44 @@ export class DashboardComponent implements OnInit, OnDestroy {
         });
   }
 
+  setLayout(value) {
+    const layout = {
+      showlegend: false,
+      font: {
+        size: 12
+      },
+      dragmode: 'false',
+      xaxis: {
+        fixedrange: true,
+        tickformat: '%H:%M:%S',
+        type: 'date',
+        title: {
+          font: {
+            size: 14,
+            color: '#7f7f7f'
+          }
+        }
+      },
+      yaxis: {
+        fixedrange: true,
+        rangemode: 'nonnegative',
+        range: [0, 8]
```

Comment:
cannot we make it `range: value !== 0 ? [0, 8] : []`

BTW, what is this value here?

Reply:
> cannot we make it range: value !== 0 ? [0, 8] : []

No, we can't.

> BTW, what is this value here?

This value is nothing but the range of y-axis when there is no value on y-axis (i.e. - [0, 0, 0, 0]), just to make tick difference in non-floating number.
Otherwise, when data on y-axis is [0, 0, 0, 0], it means the range is [0, 1] and because of this range, tick difference will be in floats number.